scripts/smoke_test.py

```diff
@@ -0,0 +1,99 @@
+import os
+import subprocess
+import time
+
+import argh
+import requests
+import yaml
```

Comment:
why not just write this as a go script? it wouldn't be absurd to add it as cli entry point (or as a thing to curator?)

argh/requests require dep management of some sort,

Reply:
Closing this PR in favor of a cli go solution.